workos/typing/literals.py

```diff
@@ -11,8 +11,11 @@
 
 
 def convert_unknown_literal_to_untyped_literal(
-    value: Any, handler: ValidatorFunctionWrapHandler, info: ValidationInfo
-) -> Union[LiteralString, UntypedLiteral]:
+    value: Any,
+    handler: ValidatorFunctionWrapHandler,
+    info: ValidationInfo,
+    # TODO: Find a way to better type, give that the last case in the except block truly can return Any
+) -> Union[LiteralString, UntypedLiteral, Any]:
```

Comment:
Do you think pyndantic's own examples don't hold up to strict checking? https://docs.pydantic.dev/latest/concepts/validators/#before-after-wrap-and-plain-validators